Microsoft/microsoft-intune/ingest/parser.yml

```diff
@@ -21,9 +21,14 @@ stages:
           action.target: 'user'
           action.type: '{{json_event.message.category}}'
           event.kind: 'event'
-          event.type: ['info']
+          event.type: [ 'info' ]
           host.id: '{{json_event.message.properties.DeviceId}}'
+
+      - set:
           host.ip: '{{json_event.message.properties.WifiMacAddress}}'
+        filter: '{{json_event.message.properties.WifiMacAddress | is_ipaddress}}'
```

Comment:
According to the name of the source field, we maybe should remove this ECS field (the value is copied in the following line).